Pacemaker/Pacemaker-2-changelog.md

```diff
@@ -0,0 +1,245 @@
+- [Differences between Pacemaker 1.1 and 2.0](#org31d2d10)
+  - [Generic points:](#orgbd42bfc)
+    - [2.0 breaks rolling upgrades from Pacemaker 1.1.10 and earlier on corosync 2 and greater (and of course from any version on other stacks)](#orga3109e6)
+    - [Only 2.0 will be actively developed from this point, but at least some bug fixes will be backported to the 1.1 branch for the foreseeable future, and we will eventually do at least one more 1.1 release with those fixes](#org324eeee)

```

Comment:
Any idea about end of life date for pacemaker 1.1?

Reply:
I didn't manage to find any information or official statement about that.
Asked a question in user mail group. Waiting for reply.

Reply:
According to reply I received the Pacemaker 1.1 support is already over.
Details: https://lists.clusterlabs.org/pipermail/users/2021-February/028423.html